lib/cucumber/core/gherkin/document.rb

```diff
@@ -3,11 +3,12 @@ module Cucumber
   module Core
     module Gherkin
       class Document
-        attr_reader :uri, :body
+        attr_reader :uri, :body, :language
 
-        def initialize(uri, body)
-          @uri = uri
-          @body = body
+        def initialize(uri, body, language=nil)
```

Comment:
That's because there's a point where the language gets passed in as `nil` (as opposed to just being omitted) and that overrode the default. So inside the initialize method, it conditionally sets it to `en` if it's `nil`.